pandas/info.py

```diff
@@ -1,20 +0,0 @@
-"""
-pandas - a powerful data analysis and manipulation library for Python
-=====================================================================
-
-See http://pandas.pydata.org/ for full documentation. Otherwise, see the
```

Comment:
Yes, you can just put the exact same string on the first lines of `__init__.py`, and it will be interpreted as the module docstring.

I would put just the same as here (title, and link to full docs). In principle we could do some clever parsing of a README file and adding that, but I wouldn't do that effort personally.